README.md

```diff
@@ -189,9 +194,15 @@ bundle install --jobs $(nproc) --with development test --path vendor/bundle
 bundle exec yarn install --frozen-lockfile --non-interactive
 export SECRET_KEY_BASE=$(bundle exec rails secret)
 bundle exec rails assets:precompile
-# somehow generate config/master.key
+# removing the credentials.yml.enc file is required unless you were given the matching master.key by the developers
```

Comment:
todo for myself: we should refactor this a bit so credentials for the [voxpupu.li](https://voxpupu.li) isn't stored in this repo.